schematic/src/gschem_about_dialog.c

```diff
@@ -65,23 +65,48 @@ void about_dialog (GschemToplevel *w_current)
     g_error_free (error);
   }
 
-  gtk_show_about_dialog (
-      GTK_WINDOW (w_current->main_window),
-      "version",        version_string,
-      "logo",           logo,
-      "title",          _("About gschem"),
-      "comments",       _("gEDA: GPL Electronic Design Automation"),
-      "copyright",
-      /* TRANSLATORS: "ChangeLog" is a literal filename; please don't translate it. */
-      _("Copyright © 1998-2012 Ales Hvezda"
-        " <ahvezda@geda.seul.org>\n"
-        "Copyright © 1998-2012 gEDA Contributors"
-        " (see ChangeLog for details)"),
-      "website",        "http://geda-project.org/",
-      NULL);
+
+  GtkWidget* dlg = gtk_about_dialog_new();
+  GtkAboutDialog* adlg = GTK_ABOUT_DIALOG (dlg);
+
+  gtk_about_dialog_set_version (adlg, version_string);
+  gtk_about_dialog_set_logo (adlg, logo);
+  gtk_about_dialog_set_comments (adlg, _("Lepton Electronic Design Automation"));
+
+  gtk_about_dialog_set_copyright (adlg,
+    _("Copyright © 1998-2018 by Ales Hvezda and the respective original authors.\n"
```

Comment:
This line should be in line with *README*: 1998-201**7**

Reply:
IMO, the first line is about gEDA/gaf authors, and the second line about Lepton devs. So no, don't change the latter. Better fix README :-)